### PR TITLE
steven: add more splash messages, and new logo textures

### PR DESCRIPTION
**Pull request by drXor**
Now we can add new textures to the big STEVEN logo, too! Animations are also supported amazingly.

### CODE REVIEW

File: ui_logo.go [hunk 1]
```diff
@@ -46,17 +72,7 @@ var (
...
-	case 0:
-		tex = render.GetTexture("blocks/cobblestone")
-	case 1:
-		tex = render.GetTexture("blocks/netherrack")
-	case 2:
-		tex = render.GetTexture("blocks/dirt")
-	case 3:
-		tex = render.GetTexture("blocks/planks_oak")
-	}
+	var tex *render.TextureInfo = render.GetTexture(logoTexture)
```
Comment (thinkofdeath) [inline ui_logo.go]: this can be simplified to tex := render.GetTexture(logoTexture)

File: ui_logo.go [hunk 2]
```diff
@@ -171,6 +187,29 @@ var stevenLogoLines = []string{
...
+	"sudo rm -rf --no-preserve-root /",
+	fmt.Sprintf("Splash generated at %s", time.Now().Unix()),
+	"Thinkofdeath.getClass().getField(\"sanity\").set(Thinkofdeath, null);",
+	"There is no God, only Zuul",
+	"Now with potatoes!",
+	"ask :: String -> Int; ask x = 42",
+	"And then you cleanse them in a ball of atomic fire!",
+	"I'm a little matrix, square and stout,",
+	"this is my transpose, this is my count.",
+	"But I am le tired!",
```
Comment (thinkofdeath) [inline ui_logo.go]: eh don't really like this one

File: ui_logo.go [hunk 3]
```diff
@@ -171,6 +187,29 @@ var stevenLogoLines = []string{
...
+	"<?php if(\"6 geese\" + \"4 chickens\" == \"10 birds\") echo(\"lolphp\");",
+	"All hail the Cat Godess!",
+	"var f = function() { return f; };",
+	"unsafe.allocateObject(Unsafe.class);",
+	":(){ :|:& };:",
+	"You must pay to view this content.",
+	"No touchy the topic", "You will never find this splash!",
```
Comment (thinkofdeath) [inline ui_logo.go]: Why this there two on the same line here?

File: ui_logo.go [hunk 4]
```diff
@@ -37,8 +37,34 @@ type uiLogo struct {
...
+		"blocks/stone_andesite",
+		"blocks/dirt_podzol_top",
+		"blocks/portal",
+		"blocks/prismarine_rough",
+		"blocks/soul_sand",
+		"blocks/lava_still",
+		"blocks/hay_block_top",
+		"blocks/log_acacia",
+		"blocks/red_sandstone_carved",
+		"blocks/command_block",
```
Comment (thinkofdeath) [inline ui_logo.go]: Wont some of these look weird?

Comment (drXor) [inline ui_logo.go]: Test them if you want.

File: ui_logo.go (hunk 3 above)
Comment (drXor) [inline ui_logo.go]: `go fmt` decided to remove about 100 spaces that made that splash not appear in the source code at first glance. Oh well, killed the joke.

**Comment by thinkofdeath** [comment]
Pulled manually

**Pull request closed without merging**